input/resources/au-medicationadministration.xml

```diff
@@ -43,6 +43,53 @@
       <type>
         <code value="CodeableConcept"/>
       </type>
+      <binding>
+        <extension url="http://hl7.org/fhir/tools/StructureDefinition/additional-binding">
+          <extension url="purpose">
+            <valueCode value="preferred"/>
+          </extension>
+          <extension url="valueSet">
+            <valueCanonical value="https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1"/>
```

Comment:
This opens to https://tx.ontoserver.csiro.au/fhir/ValueSet/australian-medication-1

Reply:
@JayMurdoch - that's nice. What's the review comment, i.e. what has to change and why?

Reply:
I now understand this is a known issue, and no change is required. Thanks.

Reply:
Please resolve the conversation when you are satisfied.

Reply:
This comment is now resolved.